frontend/src/components/NewProject/Sync/index.jsx

```diff
@@ -6,22 +6,25 @@ import { useRouter } from 'next/router'
 import { isEmpty } from 'lodash'
 
 import { AuthContext } from '@contexts/AuthContext'
-import { mirrorRepo } from '@utils/giteaApi'
-import { SyncOp, NoOp } from '../Ops'
+import { deleteRepo, mirrorRepo } from '@utils/giteaApi'
 import { urlToName } from '@utils/index'
-import styles from './index.module.scss'
 import SyncRepoFromModel from '@models/SyncRepoForm'
 import useForm from '@hooks/useForm'
+import { SyncOp, NoOp } from '../Ops'
+import { SyncConflictModal } from '../ConflictModals'
```

Comment:
Let's not start pluralizing or component modules, even if they do contain more than one component.

Reply:
Resolved in 20a9189.